static/src/javascripts/projects/commercial/modules/hosted/video.js

```diff
@@ -73,6 +78,22 @@ define([
 
     function init() {
         return new Promise(function (resolve) {
+            var $youtubeIframe = $('.js-hosted-youtube-video');
+            $youtubeIframe.each(function(el){
```

Comment:
in that case we'd need a different selector for them, so would need to change this code anyway, so.?